### PR TITLE
Reverts Replays to 1.0 until Monster fixes it

### PR DESCRIPTION
**Pull request by JamieD1**
:cl:  
rscadd: Replays 1.0
rscdel: Replays 2.0
/:cl:

### CODE REVIEW

**Comment by Simplehorror** [comment]
watching the admins complain in the forums about how they couldn't solve complaints because replays weren't working is pathetic. do they not even know about logs anymore?

**Comment by LazennG** [comment]
> watching the admins complain in the forums about how they couldn't solve complaints because replays weren't working is pathetic. do they not even know about logs anymore?

me when there's more to a situation than logs showing who attacked who first

**Comment by Bibby0110** [comment]
> > watching the admins complain in the forums about how they couldn't solve complaints because replays weren't working is pathetic. do they not even know about logs anymore?
> 
> me when there's more to a situation than logs showing who attacked who first

Replays makes it easier but it’s definitely possible to get the gist of events with logs

**Comment by JamieD1** [comment]
> > > watching the admins complain in the forums about how they couldn't solve complaints because replays weren't working is pathetic. do they not even know about logs anymore?
> > 
> > 
> > me when there's more to a situation than logs showing who attacked who first
> 
> Replays makes it easier but it’s definitely possible to get the gist of events with logs

Good luck telling the admins that.

**Comment by cowbot92** [comment]
"Hey guys this new system is cool but it kinda doesn't work, can we use the old one that works" "God lazy fucking admins cant even be bothered to look through the 9.4 billion logs that are generated each round which gives you a vague idea of who touched what"

**Comment by Simplehorror** [comment]
> "Hey guys this new system is cool but it kinda doesn't work, can we use the old one that works" "God lazy fucking admins cant even be bothered to look through the 9.4 billion logs that are generated each round which gives you a vague idea of who touched what"

i'm literally seeing so many complaints be closed right now because "replays are broken so it's LITERALLY impossible to solve". logs aren't hard to parse, you're just used to replays

**Comment by cowbot92** [comment]
> 

And you don't understand what it takes to solve complaints sometimes. When we have a system that works there's 0 reason to use a broken version that makes everyone's lives worse.

**Comment by ynot01** [comment]
Logs only relay certain actions that coders determine are worth logging

Replays relay EVERYTHING

Things like what people saw, where they were, what they were doing in between the 2 minutes of empty logs, are extremely useful and oftentimes necessary to make a proper assessment

**Comment by Simplehorror** [comment]
> > 
> 
> And you don't understand what it takes to solve complaints sometimes. When we have a system that works there's 0 reason to use a broken version that makes everyone's lives worse.

I mean I do? I appreciate admins but please don't make out that your job is something that only certain people can do

**Comment by Simplehorror** [comment]
> Logs only relay certain actions that coders determine are worth logging
> 
> Replays relay EVERYTHING
> 
> Things like what people saw, where they were, what they were doing in between the 2 minutes of empty logs, are extremely useful and oftentimes necessary to make a proper assessment

tell me how you're going to check if someone fucked with a freezer's temperature via a replay

**Comment by neeshacark** [comment]
i challenge anyone saying 'lol just use logs' to solve a complaint that way, its not fun and even if you do it you'll only be half sure you're right. takes so much less time using replays

**Comment by Simplehorror** [comment]
jesus how do you solve tickets ingame when there's no replay? do you wait for the next round?

**Comment by TheRyeGuyWhoWillNowDie** [comment]
nobody tell him

**Comment by Bibby0110** [comment]
> > 
> 
> And you don't understand what it takes to solve complaints sometimes. When we have a system that works there's 0 reason to use a broken version that makes everyone's lives worse.

That's like saying the doctors don't need to provide aid because the computers are down

**Comment by cowbot92** [comment]
> > > 
> > 
> > 
> > And you don't understand what it takes to solve complaints sometimes. When we have a system that works there's 0 reason to use a broken version that makes everyone's lives worse.
> 
> That's like saying the doctors don't need to provide aid because the computers are down

It's fine I don't really give a shit anymore, do whatever.

**Comment by Hopekz** [comment]
> me when there's more to a situation than logs showing who attacked who first

Logs are pretty extensive. You can see people's movements and everything.

A surprising amount of logging happens

**Comment by Hopekz** [comment]
> jesus how do you solve tickets ingame when there's no replay? do you wait for the next round?

The replay is generated in real time so it is built as the round progresses so you always have an up to date replay even in the current round. That replay becomes public after the round is over for non staff.

At least that's the way it worked in 1.0 I haven't been an admin here since 2.0 dropped

**Comment by Simplehorror** [comment]
> > jesus how do you solve tickets ingame when there's no replay? do you wait for the next round?
> 
> The replay is generated in real time so it is built as the round progresses so you always have an up to date replay even in the current round. That replay becomes public after the round is over for non staff.
> 
> At least that's the way it worked in 1.0 I haven't been an admin here since 2.0 dropped

bahahahahahahahahahahahahahaha

**Comment by Hopekz** [comment]
> bahahahahahahahahahahahahahaha

It's a nice feature but unironically probably made current admins completely rely on it. 

Imo there should probably be a training requirement to have moderators show proficiency in solving cases without using replays before moving on to admin but the damage has already been done since I don't think most of the new age admins know how to do that.

**Comment by Simplehorror** [comment]
well I guess any griefers reading this now know all they have to do is do things that don't appear on replays

**Comment by Hopekz** [comment]
> well I guess any griefers reading this now know all they have to do is do things that don't appear on replays

Well, let's be honest. I'd assume it's case by case, so if something requires you to do more research like if somebody messed with a console, you can always pull up variables like hidden fingerprints without even going into the logs.

The people closing requests because of the replay viewer being down should probably be trained more to be able to do their function if the replay isn't showing what they need during the shift.
If they can't get information from replays and they're unable to gather it themselves, I assume they're just asking people what happened without realizing that people lie

At some point people will realize that these newer admins have this skill issue and abuse it.

**Comment by cowbot92** [comment]
> well I guess any griefers reading this now know all they have to do is do things that don't appear on replays

Yeah I bet you'd know all about that wouldn't you.

**Comment by JamieD1** [comment]
Mering until monster fixes the viewer
